examples/dihedral/out.json

```diff
@@ -0,0 +1,170 @@
+{
```

Comment:
rename to `dihedral.out.json` as it would otherwise be overwritten by a run. See `bend` example.